docs/reference/query-dsl/wrapper-query.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[query-dsl-wrapper-query]]
+=== Wrapper Query
+
+A query that accepts any other query as base64 encoded string.
+
+[source,js]
+--------------------------------------------------
+GET /_search
+{
+    "query" : {
+        "wrapper": {
+            "query" : "eyJ0ZXJtIiA6IHsgInVzZXIiIDogIktpbWNoeSIgfX0=" <1>
+        }
+    }
+}
+--------------------------------------------------
+// CONSOLE
+
+<1> Base64 encoded string:  `{"term" : { "user" : "Kimchy" }}`
+
+This query is more useful in the context of the high level rest client or
```

Comment:
nit: s/high level rest client/high-level REST client ?

Maybe explain why it would be useful?